__tests__/endpoints.test.js

```diff
@@ -40,4 +50,31 @@ describe('app endpoint tests', () => {
         });
     });
   });
+  describe('GET /api/articles', () => {
+    it('should respond with a status 200 and array of objects containing all articles data', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(12);
+
+          // testing individual properties on articles
+          articles.forEach((article) => {
+            expect(article).toHaveProperty('article_id', expect.any(Number));
+            expect(article).toHaveProperty('title', expect.any(String));
+            expect(article).toHaveProperty('topic', expect.any(String));
+            expect(article).toHaveProperty('author', expect.any(String));
+            expect(article).toHaveProperty('body', expect.any(String));
+            expect(article).toHaveProperty('created_at', expect.any(String));
+            expect(article).toHaveProperty('votes', expect.any(Number));
+            expect(article).toHaveProperty(
+              'article_img_url',
+              expect.any(String)
+            );
+            expect(article).toHaveProperty('comment_count', expect.any(Number));
+          });
+        });
+    });
+  });
 });
```

Comment:
Missing test for testing articles are sorted correctly, you could use jest-sorted for this: 

https://www.npmjs.com/package/jest-sorted